tests/baselines/reference/api/typescript.d.ts

```diff
@@ -8705,6 +8705,7 @@ declare namespace ts {
      * ```
      */
     function getJSDocCommentsAndTags(hostNode: Node): readonly (JSDoc | JSDocTag)[];
+    function isBinaryLogicalOperator(token: SyntaxKind): boolean;
```

Comment:
Do we want `/** @internal */` on this? (I know it's a draft but before I forget)